Zscaler/zscaler-zia/_meta/smart-descriptions.json

```
@@ -0,0 +1,72 @@
+[
+  {
+    "value": "Threat {zscaler.zia.threat.name} with category {zscaler.zia.threat.category}, class {zscaler.zia.threat.class} and event id {zscaler.zia.event_id} detected",
+    "conditions": [
+      {
+        "field": "zscaler.zia.threat.name"
+      },
+      {
+        "field": "zscaler.zia.threat.category"
+      },
+      {
+        "field": "zscaler.zia.threat.class"
+      },
+      {
+        "field": "zscaler.zia.threat.class"
+      }
+    ]
+  },
```

I suggest having one description per type of events

E.g. for audit events:
```suggestion
  {
    "value": "{source.ip} accesssed resource {zscaler.zia.resource}",
    "conditions": [
      {
        "field": "event.dataset",
	"value": "audit"
      },
      {
        "field": "source.ip"
      },
      {
        "field": "zscaler.zia.resource"
      }
    ]
  },
```